TestingLowerBounds/ForMathlib/Integrable_of_empty.lean

```diff
@@ -0,0 +1,27 @@
+import Mathlib.MeasureTheory.Integral.Bochner
+
+open MeasureTheory
+
+open Real MeasureTheory Filter MeasurableSpace
+open scoped ENNReal NNReal Topology BigOperators
+
+--TODO: put this in mathlib, maybe just after:
+#check Integrable.of_finite
+
+@[simp]
+lemma Integrable.of_empty {α β : Type*} [MeasurableSpace α] [NormedAddCommGroup β] [IsEmpty α]
+    (f : α → β) (μ : Measure α) : Integrable f μ := Integrable.of_finite μ f
+
+@[simp]
+lemma Integrable.of_empty_codomain {α β : Type*} [MeasurableSpace α] [NormedAddCommGroup β] [IsEmpty β]
+    {f : α → β} {μ : Measure α} : Integrable f μ := by
```

Comment:
The name should use `isEmpty`:
```suggestion
lemma Integrable.of_isEmpty_codomain {α β : Type*} [MeasurableSpace α] [NormedAddCommGroup β]
    [IsEmpty β] {f : α → β} {μ : Measure α} : Integrable f μ := by
```

Reply:
Also why are the arguments of this one implicit, while `Integrable.of_isEmpty` has explicit arguments?

Reply:
Right, this was a mistake, I think they should all have explicit arguments